src/js/page/ui/settings.js

```diff
@@ -46,7 +46,7 @@ export default class Settings {
     // Prevents bounce effect on desktop.
     // Firefox uses DELTA_LINE on a mouse wheel, ignore it
     // 0 is "pixels"
-    if (!event.deltaMode) {
+    if (event.deltaMode === 0) {
```

Comment:
Just to confirm, I don't suppose this can be undefined or null? My understanding is that this is a Firefox-specific workaround where `deltaMode` is `0`.

Reply:
It's always a number, yeah